### ISSUE TITLE
how to get some testnet xem coin? I can't find any faucet

### ISSUE THREAD

**Issue by harryweiyongjie** [nemtech/nem2-docs#290]
**Describe the error**
A clear and concise description of what the error is.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by dgarcia360** [comment]
@harryweiyongjie There is not currently a testnet publicly available using Catapult. What you could do to test NEM's next core engine is to run your own private network for testing and development purposes: https://nemtech.github.io/getting-started/setup-workstation.html

**Comment by harryweiyongjie** [comment]
> @harryweiyongjie There is not currently a testnet publicly available using Catapult. What you could do to test NEM's next core engine is to run your own private network for testing and development purposes: https://nemtech.github.io/getting-started/setup-workstation.html

emmmm.....all right, I just deploy the testnet, and just need some coin to test.Another question: now the network is running by NIS node, I certainly develop with NIS, do I have to turn to Catapult?

**Comment by dgarcia360** [comment]
Hi @harryweiyongjie,

> I just deploy the testnet, and just need some coin to test.

* Catapult: If you have deployed the network with catapult-service-bootstrap, you can find the account keys loaded with ``cat.currency`` [here](https://nemtech.github.io/getting-started/setup-workstation.html?code=typescript#getting-a-test-account). 

* NIS1: Share your address and the amount of nem:xem requested in the [forum](https://forum.nem.io/t/paste-you-address-here-for-beta-nem-testnet-xem/829/2325). 

> now the network is running by NIS node, I certainly develop with NIS, do I have to turn to Catapult?

From [NIP-8](https://github.com/nemtech/NIP/blob/master/NIPs/nip-0008.md#ellected-situation):
 
> Catapult technology will be released alongside of NIS1, resulting in two chains and networks with Snapshot scheme.

In other words, while there are nodes running NIS1, the chain will continue existing. That means you can opt to migrate or continuing using the current network. If you decide to migrate your app from NIS1 to Catapult, I advise you too read what has changed: https://nemtech.github.io/guides/migration/migrating-from-nis1-to-catapult.html

**Comment by harryweiyongjie** [comment]
oh，got it.  Thanks for your help~

**Comment by harryweiyongjie** [comment]
> Hi @harryweiyongjie,
> 
> > I just deploy the testnet, and just need some coin to test.
> 
> * Catapult: If you have deployed the network with catapult-service-bootstrap, you can find the account keys loaded with `cat.currency` [here](https://nemtech.github.io/getting-started/setup-workstation.html?code=typescript#getting-a-test-account).
> * NIS1: Share your address and the amount of nem:xem requested in the [forum](https://forum.nem.io/t/paste-you-address-here-for-beta-nem-testnet-xem/829/2325).
> 
> > now the network is running by NIS node, I certainly develop with NIS, do I have to turn to Catapult?
> 
> From [NIP-8](https://github.com/nemtech/NIP/blob/master/NIPs/nip-0008.md#ellected-situation):
> 
> > Catapult technology will be released alongside of NIS1, resulting in two chains and networks with Snapshot scheme.
> 
> In other words, while there are nodes running NIS1, the chain will continue existing. That means you can opt to migrate or continuing using the current network. If you decide to migrate your app from NIS1 to Catapult, I advise you too read what has changed: https://nemtech.github.io/guides/migration/migrating-from-nis1-to-catapult.html

oh，got it. Thanks for your help~
